docs/admin/components.mdx

```diff
@@ -19,21 +19,24 @@ To swap in your own React component, first, consult the list of available compon
 
 You can override a set of admin panel-wide components by providing a component to your base Payload config's `admin.components` property. The following options are available:
 
-| Path                  | Description  |
-| --------------------- | -------------|
-| **`Nav`**             | Contains the sidebar and mobile Nav in its entirety. |
-| **`BeforeDashboard`** | Array of components to inject into the built-in Dashboard, _before_ the default dashboard contents. |
-| **`AfterDashboard`**  | Array of components to inject into the built-in Dashboard, _after_ the default dashboard contents. [Demo](https://github.com/payloadcms/payload/tree/master/test/admin/components/AfterDashboard/index.tsx)|
-| **`BeforeLogin`**     | Array of components to inject into the built-in Login, _before_ the default login form. |
-| **`AfterLogin`**      | Array of components to inject into the built-in Login, _after_ the default login form. |
-| **`BeforeNavLinks`**  | Array of components to inject into the built-in Nav, _before_ the links themselves. |
-| **`AfterNavLinks`**   | Array of components to inject into the built-in Nav, _after_ the links. |
-| **`views.Account`**   | The Account view is used to show the currently logged in user's Account page. |
-| **`views.Dashboard`** | The main landing page of the Admin panel. |
-| **`graphics.Icon`**   | Used as a graphic within the `Nav` component. Often represents a condensed version of a full logo. |
-| **`graphics.Logo`**   | The full logo to be used in contexts like the `Login` view. |
-| **`routes`**          | Define your own routes to add to the Payload Admin UI. [More](#custom-routes) |
-| **`providers`**       | Define your own provider components that will wrap the Payload Admin UI. [More](#custom-providers) |
+| Path                        | Description  |
+| ---------------------       | -------------|
+| **`Nav`**                   | Contains the sidebar and mobile Nav in its entirety. |
+| **`Logout.Component`**      | A custom React component. |
```

Comment:
This is just semantics, but we typically will only capitalize a property if _**it itself is a React component.**_ So in this case, `Logout` is not a React component - rather, it's a group of React components. So it should be lowercase. 

And I'd actually suggest to be more explicit with your naming for `Component` as well. 

So with these changes, how about we go with `logout.Button`?

Reply:
documentation updated.